src/screens/Home/OnboardingScreen/StepScreens.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react'
+import {
+  NavigationScreenProp, 
+  NavigationState
+} from 'react-navigation'
+import {
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+} from 'react-native'
+import styles from './stepStyles'
+
+/* AKA: Q&A screen */
+export default function OnboardingScreen(props: Props) {
+  const {navigate} = props.navigation
+  const [step, setStep] = useState(2)
+  const [text, setText] = useState('Select questions to answer')
+  const [header, setHeader] = useState('Step 1')
+  const [textStyle, setTextStyle] = useState(styles.stepOne)
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.main}>
+        <Text style={styles.header}>{header}</Text>
+        <Text style={textStyle}>{text}</Text>
+        <Image style={styles.image} resizeMode='contain' source={require('../../../../assets/images/rectangle.png')} />
+        <Image style={styles.image} resizeMode='contain' source={require('../../../../assets/images/rectangle.png')} />
+        <Image style={styles.image} resizeMode='contain' source={require('../../../../assets/images/rectangle.png')} />
+      </View>
```

Comment:
Not sure if you didn't get to this yet, but you don't update the actual content of the screens, only the text and title. 
https://www.figma.com/file/MQJ8XLALQDMzBaI5yMgGAB/Gladeo?node-id=0%3A1